src/Features/LanguageServer/Protocol/IRefreshListener.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.LanguageServer
+{
+    internal interface IRefreshListener : IDisposable
```

Comment:
I think this was probably my mistake for not being clear - what I meant was a wrapper interface around the jsonrpc object that has a SendNotification method that takes in the params and method name.

Reply:
Done! Hopefully the abstraction makes things easier in the future. I wasn't sure about naming, if you have any suggestions feel free to lmk